packages/generator/src/vdm-types.ts

```diff
@@ -117,21 +117,27 @@ export type VdmActionFunctionImportReturnType =
   | VdmFunctionImportReturnType;
 
 export interface VdmFunctionImportReturnType {
-  builderFunction: string;
+  builderFunction?: string;
   returnType: string;
   /**
    * @deprecated Since v1.22.0. Use `isCollection` instead.
    */
   isMulti?: boolean;
   isCollection: boolean;
   returnTypeCategory: VdmReturnTypeCategory;
+  unsupportedFunction?: VdmUnsupportedFunction;
```

Comment:
The name of the property, as well as the name of the enum imply that this represents a function or a type of function. You called it "reason" yesterday. How about `neverReason` (or `unsupportedReason`)?